### PR TITLE
Added serialize function to activations.py

### PR DESCRIPTION
**Pull request by Nik-code**
## PR Description
- Modified file: https://github.com/unifyai/ivy/blob/main/ivy/functional/frontends/tensorflow/keras/activations.py
- Added a serialize function to convert activation functions back into their string identifiers.
- Ensured compatibility with supported TensorFlow versions and data types.
## Related Issue
Closes https://github.com/unifyai/ivy/issues/1569

### CODE REVIEW

**Comment by Nik-code** [comment]
@paulaehab Hey, can you please review my PR. Thank you.

**Comment by paulaehab** [comment]
Hello @Nik-code  first of all thanks for contributing but could you please add test for the fucntion you will find more details in this link : (https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)

**Comment by Nik-code** [comment]
Hey @paulaehab , I have added the relevant tests for the serialize function in the _test_activation.py_ file. I've added them with compliance to the documentation you mentioned [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html), please review the new commit for the changes, thanks.

**Comment by abdul-rehman18** [comment]
https://github.com/unifyai/ivy/issues/25772

File: ivy/functional/frontends/tensorflow/keras/activations.py [hunk 1]
```diff
@@ -111,6 +111,44 @@ def selu(x):
     return ivy.selu(x)
 
 
+@with_supported_dtypes(
+    {"2.13.0 and below": ("float16", "float32", "float64")},
+    "tensorflow",
+)
+def serialize(func, custom_objects=None):
```
Comment (paulaehab) [inline ivy/functional/frontends/tensorflow/keras/activations.py:118]: we need to adhere the Tensorflow frontend names for arguments it was in documentation (activation, use_legacy_format=False) you could have a look here (https://www.tensorflow.org/api_docs/python/tf/keras/activations/serialize )

**Comment by abdul-rehman18** [comment]
https://github.com/unifyai/ivy/pull/25790

@paulaehab 
kindly review it

**Comment by paulaehab** [comment]
Hey @Nik-code  could you please follow the instruction of this page https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues to add issue related to serialize function in https://github.com/unifyai/ivy/issues/1569

**Comment by abdul-rehman18** [comment]
@paulaehab 
I also made a pr

**Comment by paulaehab** [comment]
> #25790
> 
> @paulaehab kindly review it

Hello @abdul-rehman18 I am very sorry @Nik-code  had already made the function and he just forget to add issue with function name , also it was my fault that I didn't get it I am sorry for that could you pick another function and again sorry for this mistake

**Comment by abdul-rehman18** [comment]
no problem sir @paulaehab 
Thanks

**Comment by paulaehab** [comment]
> no problem sir @paulaehab Thanks
Thanks for your understanding and again sorry for this mistake if you make another PR feel free to ask review for me

**Comment by Nik-code** [comment]
Hey @paulaehab ,  thank you for the suggestions. I'll make sure to make the requested changes soon and post the commit 😊

**Comment by Nik-code** [comment]
Close #25851

**Comment by Nik-code** [comment]
Hey @paulaehab I have made a new commit changing the function description according to the TensorFlow documentation, Also I have correctly created a sub-task which I mentioned above that is now referenced in #1569

**Comment by paulaehab** [comment]
Hey @Nik-code  thanks for contributing look great for me 😄

**Pull request merged**